CODE/VulnScan/v2-deprecated/_train.py

```
@@ -0,0 +1,429 @@
+
+# ---------------------------------------
+# UTILITIES
+# ---------------------------------------
+
+
+@deprecated(reason="This function is used to load data from a directory. Its for training v2 models, which is now deprecated, use train.py v3 instead.", removal_version="3.3.0")
+def load_data(data_dir: str) -> tuple[list[str], np.ndarray]:
+    """Loads text data and labels from the directory."""
+    texts, labels = [], []
+    for file_name in os.listdir(data_dir):
+        with open(os.path.join(data_dir, file_name), "r", encoding="utf-8") as f:
```

Ruff (0.8.0)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)